doc/source/theory/dem/dem.rst

```diff
@@ -121,54 +127,89 @@ Where:
 
 * :math:`\nu_i, \nu_j` poisson coefficient of particle i or j;
 
-Rolling friction may be calculated through a constant torque model or a viscous torque model corresponding to those equations:
+~~~~~~~~~~~~~~~~~~~~
+Coulomb's limit
+~~~~~~~~~~~~~~~~~~~~
+
+Coulomb's criterion is violated when the following condition is not respected during a collision:
 
 .. math::
-    \mathbf{M}_{ij}^{r} &= -\mu_{r}R_{e}|\mathbf{F}_{ij}^{n}| \mathbf{\hat{\omega}}_{ij} \\
-    \mathbf{M}_{ij}^{r} &= -\mu_{r}R_{e}|\mathbf{F}_{ij}^{n}||\mathbf{V}_{\omega}| \mathbf{\hat{\omega}}_{ij}
+    |\mathbf{F}_{ij}^{t}| \geq \mu |\mathbf{F}_{ij}^{n}|
 
-Where the parameters are:
+
+A violation means the collision is having gross sliding and tangential force needs to be limited to the Coulomb limit.
+To do so, the tangential overlap :math:`\mathbf{\delta_t}` is first limited and then the tangential force is recalculated.
+
+The tangential overlap is calculated with the tangential force with no damping force as default nonlinear contact model as follow:
```

Comment:
I have an other PR coming about the tangential overlap and the Coulomb  limit. Do you think this could wait for the PR? 

It's weird syntax and it will become change in the code...

Reply:
okidoke